crd-docs/crd/kiali.io_kialis.yaml

```diff
@@ -779,6 +779,9 @@ spec:
                       root_namespace:
                         description: "The namespace to treat as the administrative root namespace for Istio configuration."
                         type: string
+                      skip_wildcard_gateway_hosts:
+                        description: "To skip KIA0301 validation for Gateways with wildcards in hosts."
```

Comment:
This needs a lot more detail to explain what this is all about. It is confusing because one wonders why I can't just add KIA301 to the ignore list. So explain what this setting does and when you would want to set it.